packages/@aws-cdk/aws-sns/lib/subscription.ts

```diff
@@ -41,6 +43,14 @@ export interface SubscriptionOptions {
    * @default - the region where the CloudFormation stack is being deployed.
    */
   readonly region?: string;
+
+  /**
+   * Queue used when `deadLetterQueue` is enabled.
+   * Default queue is created when not specified.
```

Comment:
Docs here still need to be updated since we no longer build a default queue.
```suggestion
   * Queue to be used as dead letter queue. If not passed no dead letter queue is enabled.
```